sipa/templates/base.html

```diff
@@ -56,10 +56,8 @@
         <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/bootstrap-social.css') }}"/>
         <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/style.css') }}"/>
 
-        <script>
-            function get_language() {
-                return "{{ get_locale() }}";
-            }
+        <script type="application/json" id="locale">
+            {{ get_locale() | string | tojson }}
```

Comment:
I suggest, that you store the different parts of the locale in a dictionary, instead of just the locale identifier:

```jinja
<script type="application/json" id="locale">
    {% set locale = get_locale() %}
    {{ {'language': locale.language, 'territory': locale.territory, 'script': locale.script, 'variant': locale.variant} | tojson }}
</script>
```